static/templates/settings/organization_permissions_admin.hbs

```diff
@@ -191,18 +191,37 @@
             </div>
             <div class="input-group">
                 <label for="realm_edit_topic_policy" class="dropdown-title">{{t "Who can move messages to another topic" }}</label>
-                <select name="realm_edit_topic_policy" id="id_realm_edit_topic_policy" class="prop-element" data-setting-widget-type="number">
+                <select name="realm_edit_topic_policy" id="id_realm_edit_topic_policy" class="prop-element move-message-policy-setting" data-setting-widget-type="number">
                     {{> dropdown_options_widget option_values=edit_topic_policy_values}}
                 </select>
             </div>
 
             <div class="input-group">
                 <label for="realm_move_messages_between_streams_policy">{{t "Who can move messages to another stream" }}
                 </label>
-                <select name="realm_move_messages_between_streams_policy" class="setting-widget prop-element" id="id_realm_move_messages_between_streams_policy" data-setting-widget-type="number">
+                <select name="realm_move_messages_between_streams_policy" class="setting-widget prop-element move-message-policy-setting" id="id_realm_move_messages_between_streams_policy" data-setting-widget-type="number">
                     {{> dropdown_options_widget option_values=move_messages_between_streams_policy_values}}
                 </select>
             </div>
+
+            <div class="input-group time-limit-setting">
+                <label for="realm_message_move_limit_seconds" class="dropdown-title">{{t "Time limit for editing topics and moving messages" }} <i>({{t "applies only to members and guests" }})</i></label>
+                <select name="realm_message_move_limit_seconds" id="id_realm_message_move_limit_seconds" class="prop-element" data-setting-widget-type="time-limit">
+                    {{#each msg_move_limit_dropdown_values}}
+                        <option value="{{value}}">{{text}}</option>
+                    {{/each}}
+                </select>
+                <div class="dependent-settings-block">
+                    <label for="id_realm_message_move_limit_minutes" class="inline-block realm-time-limit-label">
+                        {{t 'Duration editing is allowed after posting (minutes)'}}:&nbsp;
+                    </label>
+                    <input type="text" id="id_realm_message_move_limit_minutes"
+                      name="realm_message_move_limit_minutes"
+                      class="time-limit-custom-input"
+                      autocomplete="off"
+                      value="{{ realm_message_move_limit_seconds }}"/>
```

Comment:
How does the seconds/minutes conversion work here? It seems like the value displayed here is a number of seconds, even though the input field name is a number of minutes.

Reply:
I have removed this line and we instead set the value in JS since we already set the dropdown value and visibility of custom input in JS. It also helps in live-update code.